.github/workflows/ci.yml

```diff
@@ -151,23 +151,42 @@ jobs:
       - name: Build packages
         run: npm run build --workspace @puppeteer-test/test
       - name: Setup cache for Chrome binary
+        if: ${{ matrix.suite != 'chrome-bidi' }}
         uses: actions/cache@v3
         with:
           path: ~/.cache/puppeteer/chrome
           key: ${{ runner.os }}-Chrome-${{ hashFiles('packages/puppeteer-core/src/revisions.ts') }}-${{ hashFiles('packages/puppeteer/src/node/install.ts') }}
       - name: Install Chrome
+        if: ${{ matrix.suite != 'chrome-bidi' }}
         run: npm run postinstall
+      - name: Setup cache for Chrome Canary binary
+        if: ${{ matrix.suite == 'chrome-bidi' }}
+        uses: actions/cache@v3
+        with:
+          path: ~/.cache/puppeteer/chrome-canary
+          key: ${{ runner.os }}-Chrome-Canary-${{ hashFiles('package.json') }}
+      - name: Install Chrome Canary
+        if: ${{ matrix.suite == 'chrome-bidi' }}
+        id: browser
+        run: |
+          bidi_chrome_version=$(cat ./node_modules/chromium-bidi/.browser) &&
+          executable_path=$(cut -d' ' -f2 <<< $(npx @puppeteer/browsers install $bidi_chrome_version --path ~/.cache/puppeteer/chrome-canary)) &&
```

Comment:
I do not think -f2 is correct, probably it as to be `-f2-` as the executable path might have spaces

Reply:
I think `-f2` is correct as my
```bash
echo $(cut -d'.' -f2- <<<  '.s.2..') --> s.2..
echo $(cut -d'.' -f2 <<<  '.s.2..') --> s
```